src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentQueue.cs

```diff
@@ -627,6 +627,18 @@ public bool TryPeek(out T result)
             return false;
         }
 
+        /// <summary>
+        /// Removes all objects from the <see cref="ConcurrentQueue{T}"/>.
+        /// </summary>
+        public void Clear()
+        {
+            while (!IsEmpty)
+            {
+                Segment head = _head;
+                head.Clear();
```

Comment:
I added comment and created tests. But I have issues with config files to add API correctly.